example/README.md

```diff
@@ -227,38 +229,40 @@ In the event that you are running the tests outside of the Cumulus sandbox envir
 
 ### Adding tests for an existing workflow
 
-Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and supporting JSON files can go in there.
+Workflow tests are located in the `/spec/parallel` or `/spec/serial` directory depending on whether or not they can run in parallel or not. Any tests and supporting JSON files can go in there.
 
 ### Adding a new test workflow
 
-The workflow should be configured as it would be for a normal Cumulus deployment in a workflows yaml file. It must be deployed to the current deployment if testing locally.
+The workflow should be configured as it would be for a normal Cumulus deployment in a workflows terraform file. It must be deployed to the current deployment if testing locally.
 
-The workflows yaml files are located in the `/workflows/` folder and are split up to make the workflows easier to find and understand. When adding a new file, make sure to update the `app/config.yml` `stepFunctions` field.
+The workflows terraform files are located in the `/example/cumulus-tf` directory and are split up to make the workflows easier to find and understand.
 
-A new folder should be added in the `/spec` folder for the workflow and the tests should go into that folder with the input JSON files.
+A new directory should be added in the `/spec` directory for the workflow and the tests should go into that directory with the input JSON files.
```

Comment:
Can we re-word this somehow? Since the directories aren't being added directly to `spec/`

Reply:
Yeah, I re-wrote it, let me know what you think https://github.com/nasa/cumulus/pull/1972/commits/f24fb2d987e0d419e22242c6df6d264034b43b68